src/pinecone_client/vector_db.py

```diff
@@ -0,0 +1,54 @@
+import pinecone
+from langchain.embeddings.openai import OpenAIEmbeddings
+from dotenv import load_dotenv
+import os
+from langchain.vectorstores import Pinecone
+
+# 　環境変数の読み込み
+load_dotenv()
+INDEX_NAME = os.getenv("INDEX_NAME")
+PINECONE_API_KEY = os.getenv("PINECONE_API_KEY")
+PINECONE_ENV = os.getenv("PINECONE_ENV")
+
+if not INDEX_NAME:
+    raise ValueError("INDEX_NAME 環境変数が設定されていません。")
+if not PINECONE_API_KEY:
+    raise ValueError("PINECONE_API_KEY 環境変数が設定されていません。")
+if not PINECONE_ENV:
+    raise ValueError("PINECONE_ENV 環境変数が設定されていません。")
+
+# pinecone clientの初期化
+pinecone.init(api_key=PINECONE_API_KEY, environment=PINECONE_ENV)
+
+# インデックス初期化
+index = pinecone.Index(INDEX_NAME)
+
+# embeddingsの初期化
+model_name = "text-embedding-ada-002"  # TODO env化
+embeddings = OpenAIEmbeddings(model=model_name)
+
+# TODO:以下の関数をクラスにまとめて、重複している共通データをインスタンス変数とかにする
+
+
+def create_index_or_pass():
+    # pyrightでエラー出るのでつけてます。#TODO:refactor
+    if not INDEX_NAME:
+        raise ValueError("INDEX_NAME 環境変数が設定されていません。")
+
+    # すでに作成しているインデックスネームのリストを取得して存在確認
+    index_name_list = pinecone.list_indexes()
+
+    if INDEX_NAME not in index_name_list:
```

Comment:
```suggestion
  is_include_index = INDEX_NAME in index_name_list ## Boolに変換
    if is_include_index is  True:
```
おそらく、リーダブルコードにも記載があったと思うのですが`if文`を使用する際にはなるべく肯定形で判断できるかを検討してみてください！